Makefile

```diff
@@ -258,7 +258,11 @@ ensure:
 
 # Should run all needed commands before any PR is sent out.
 .PHONY: ready-pr
-ready-pr: lint manifests resource-docs ensure fmt
+ready-pr: lint manifests ensure fmt
+
+# Should run all needed commands to prepare a release.
+.PHONY: release-check
+release-check: resource-docs
```

Comment:
We will extend this list to 1. add release note. 2. verify release tags. ...

Reply:
You can also add it to https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/dev/ci/presubmits/validate-generated-files

Reply:
Actually I guess not, because we only want to run this pre-release!

It might be easier to just auto-generate a basic resource-doc as part of promotion, but .. not a blocker.